samples/error-correction/syndrome/syndrome.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""
+Script for running Pseudo-syndrome example as defined in Syndrome.qs.
+This script takes the number of data qubits as input.
+
+Example usage:
+    python syndrome.py -q 5
+"""
+
+import numpy as np
+import qsharp
+import argparse
+
+from Microsoft.Quantum.Samples.Hardware.Syndrome import SamplePseudoSyndrome
+
+parser = argparse.ArgumentParser(
+    prog="PseudoSyndrome",
+    description="Program for running a PseudoSyndrome circuit that detects random errors on qubits in gate sequence."
+)
+parser.add_argument("-q", "--qubits", type=int, default=1)
+
+if __name__ == "__main__":
+    args = parser.parse_args()
+    paulis = ["PauliX", "PauliY", "PauliZ"]
+    indexes = list(range(args.qubits))
+    np.random.shuffle(indexes)
```

Comment:
What is the reason to shuffle `indexes` if you are already choosing each of the `input_values` and `encoding_bases` at random?

Reply:
It shuffles the order in which the controlled Paulis are applied. In this case I agree that it doesn't really matter indeed but I thought this would be relevant in for when I have a set of fixed couplings between qubits in IRL hardware. @apaetz thoughts?

Reply:
Oh, thanks, I somehow missed that! In that case, perhaps there could be a more descriptive name for this than `indexes`?

Reply:
Good call! Also `indexes` should be renamed to `indices` at the very least :)

Reply:
You may find the `qsharp.Pauli.sample()` method interesting: https://github.com/microsoft/iqsharp/blob/master/src/Python/qsharp-core/qsharp/types.py#L35